### ISSUE TITLE
Does it work on Nvidia Macbook Pro (11,5)?

### ISSUE THREAD

**Issue by trusktr** [papyros/papyros-shell#151]
I'm curious if you might know if the UI works on Macbook Pro 11,x with Nvidia. I'm not having luck with Gnome desktop. It has the "Oh no, something went wrong" message, although it works (slowly) with the nouveau driver. Cinnamon on the other hand is buttery with the nvidia driver.

I ended up here at papyros because I've been looking around to see if there's any good desktop that works nicely with the nvidia drivers on my Macbook Pro. I'm tired of having to deal with nouveau slownes and nvidia glitches.

Perhaps Qt is better suited to Nvidia? Just curious to get some insight before I dive in only to find my nvidia driver isn't working on my 11,5 with Papyros.
##

**Comment by 29jm** [comment]
Until the proprietary Nvidia driver supports Wayland, you'll need nouveau. Perhaps there's a wiki page for your MacBook on the Arch wiki that could help you though.

**Comment by trusktr** [comment]
Yeah, I've tried it all. Nouveau works, but is just so slow. ALso, Chromium keeps freezing nouveau, so I've settled with being limited to MATE or Cinnamon (no GNOME) in order for things not to crash.
